### ISSUE TITLE
libthumbor is not being detected as a python3 compatible

### ISSUE THREAD

**Issue by jklemm** [thumbor/libthumbor#35]
Hi,

I'm confusing about py3 support on this project.

### Check python compatibility version

The site [Can I use Python3](https://caniusepython3.com/project/libthumbor) indicates that libthumbor is blocking me to go to python3:

- libthumbor (which is blocking django-thumbor)

Reading the issues, the issue #18 release a new version that is compatible with py3.

The site uses the `setup.py` info to know if the project is or not py3 compatible.
This file contains `trove classifiers` and once python 2.6 is indicated to be compatible.

In summary, we need to add the 3.x version in this file, to the project really be detected as a py3 compatible project.

### Contributing Guidelines

Looking the Contributing Guidelines, I found this:

`You'll also need python >= 2.6 and < 3.0.`

Again, I assume that anyone forgot to change this information.

Thanks!

**Comment by jklemm** [comment]
I open this issue, trying to help me and others, but this is not works.
Will close this issue, and search other library.

Thanks.
